app/api/chat/route.ts

```diff
@@ -5,74 +5,72 @@ import { createClient } from '@supabase/supabase-js'
 
 export const runtime = 'edge'
 
-  const apiUrl = process.env.API_URL;
+const apiUrl = process.env.API_URL
 
-  export async function POST(req: Request) {
-    const json = await req.json()
-    const { messages } = json
-    const message = (messages[messages.length - 1]).content
-    const userId = (await auth())?. user?.id
-    const temperature = '1.2'
-    const model = "gpt-3.5-turbo"
-    const version = "1.0.0"
-    const source = "webapp"
-  
-    if (!userId) {
-      return new Response('Unauthorized', {
-        status: 401
-      })
-    }
-  
-    const response = await fetch(`${apiUrl}/chat`, {
-      headers: {
-        'Content-Type': 'application/json',
-      },
-      method: 'POST',
-      body: JSON.stringify({
-        message,
-        temperature,
-        model
-      })
-    });
+export async function POST(req: Request) {
+  const selectedModel = window.localStorage.getItem('model')
```

Comment:
what happens when no value is in local storage?

Reply:
by default it will take 3.5